php/boot-phar.php

```diff
@@ -1,5 +1,8 @@
 <?php
 
+// Store the path to the Phar early on for `Utils\phar-safe-path()` function.
+define( 'WP_CLI_PHAR_PATH', getcwd() );
```

Comment:
Ah, I should add: the `getcwd()` produces the path to the Phar, not to a file from within the Phar.